AllWeShare/src/Controller/Front/PostController.php

```diff
@@ -34,6 +40,21 @@ public function new(Request $request, PostRepository $postRepository): Response
             return $this->redirectToRoute('post_index');
         }
 
+        if($formReport->isSubmitted() && $formReport->isValid()){
+            $user = $this->getUser();
+            //dump( $request->request ); die;
+            $postReportedId = $request->request->get('id');
+            $postReported = $postRepository->find($postReportedId);
+            $report->setPost($postReported);
+            $report->setReporter($user);
+            //dump( $report ); die;
+            $entityManager = $this->getDoctrine()->getManager();
+            $entityManager->persist($report);
+            $entityManager->flush();
+
+            $this->addFlash('success', 'Thanks you for your report');
+        }
+
         return $this->render('Front/post/index.html.twig', [
             'post' => $post,
             'form' => $form->createView(),
```

Comment:
here you need to export your 'formReport' with createView method (as you did below)